cpp/src/tsne/distances.cuh

```diff
@@ -95,9 +95,8 @@ void get_distances(const raft::handle_t &handle,
     input.indptr, input.indices, input.data, input.nnz, input.n, input.d,
     input.indptr, input.indices, input.data, input.nnz, input.n, input.d,
     k_graph.knn_indices, k_graph.knn_dists, k_graph.n_neighbors,
-    handle.get_cusparse_handle(), handle.get_device_allocator(), stream,
-    ML::Sparse::DEFAULT_BATCH_SIZE, ML::Sparse::DEFAULT_BATCH_SIZE,
-    DEFAULT_DISTANCE_METRIC);
+    const_cast<raft::handle_t &>(handle), ML::Sparse::DEFAULT_BATCH_SIZE,
```

Comment:
Oh boy. I don't know how I missed this one- that should be a `const` on the raft side for sure.